packages/teuchos/core/test/ScalarTraits/ScalarTraits_test.cpp

```diff
@@ -328,10 +328,8 @@ int main( int argc, char* argv[] ) {
     result = testOrdinalTraits<size_t>(*out);
     if(!result) success = false;
 
-#ifdef HAVE_TEUCHOS_LONG_LONG_INT
     result = testOrdinalTraits<long long int>(*out);
     if(!result) success = false;
-#endif
 
 #ifdef HAVE_TEUCHOS___INT64
```

Comment:
FYI, the above change has a slight risk of breaking Windows builds, I suspect.  We can always go back and fix that later, though.  I want to get rid of the macro first, because it will make debugging platform-specific issues like that easier.